### PR TITLE
PMMI Side of Inquiry Form Update

### PR DESCRIPTION
**Pull request by Shinsina**
Screen Shot 2021-05-20 at 4 11 31 PM
Screen Shot 2021-05-20 at 4 18 51 PM

### CODE REVIEW

**Comment by Shinsina** [comment]
[How to solve for Mundo Leaders entries in Spanish and English](https://trello.com/c/oKzezqPm/50-how-to-solve-for-mundo-leaders-entries-in-spanish-and-english)

File: packages/shared/templates/content/company.marko [hunk 1]
```diff
@@ -4,6 +4,7 @@ import hierarchyAliases from "@parameter1/base-cms-marko-web/utils/hierarchy-ali
 $ const { i18n } = out.global;
 
 $ const { id, type, pageNode } = data;
+$ const lang = input.lang
```
Comment (solocommand) [inline packages/shared/templates/content/company.marko:7]: Missing semicolon

File: packages/shared/components/page-wrappers/content/leader.marko [hunk 2]
```diff
@@ -5,6 +5,7 @@ import queryFragment from "../../../graphql/fragments/content-list";
 
 $ const { site, i18n } = out.global;
 $ const { id, type, content } = input;
+$ const lang = site.config.lang || 'en'
```
Comment (solocommand) [inline packages/shared/components/page-wrappers/content/leader.marko:8]: Use double quotes in template files

**Review by solocommand — CHANGES_REQUESTED** [review]

File: packages/shared/components/page-wrappers/content/leader.marko [hunk 3]
```diff
@@ -5,6 +5,7 @@ import queryFragment from "../../../graphql/fragments/content-list";
 
 $ const { site, i18n } = out.global;
 $ const { id, type, content } = input;
+$ const lang = site.config.lang || "en"
```
Comment (solocommand) [inline packages/shared/components/page-wrappers/content/leader.marko:8]: Missing semicolon

**Review by solocommand — CHANGES_REQUESTED** [review]

**Review by solocommand — APPROVED** [review]

**Pull request merged**